DB-schema-migration.md

```diff
@@ -0,0 +1,156 @@
+# DB Schema migration for module update
+
+Copyright (C) 2016-2019 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License, Version 2.0.
+See the file ["LICENSE"](LICENSE) for more information.
```

Comment:
Can you remove the license information? The LICENSE file covers the whole repository.
See https://dev.folio.org/guidelines/contributing/#no-license-header

Reply:
Done